hack/bootstrap-investigation.sh

```
@@ -5,37 +5,26 @@ set -e
 read -p "Enter the new investigation (package) name: " INVESTIGATION_NAME
-if [[ "${INVESTIGATION_NAME}" == "" ]] ; then
+if [[ "${INVESTIGATION_NAME}" == "" ]]; then
 	echo "Investigation name cannot be empty."
 	exit 1
-elif [[ "${INVESTIGATION_NAME}" =~ [^a-zA-Z0-9_] ]] ; then
+elif [[ "${INVESTIGATION_NAME}" =~ [^a-zA-Z0-9_] ]]; then
 	echo "Investigation name must be alphanumeric."
 	exit 1
 fi
 
 read -p "Enter new investigation description: " INVESTIGATION_DESCRIPTION
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)